locale.c

```diff
@@ -7219,10 +7219,7 @@ Perl_switch_locale_context()
 
     /* Can't use pTHX, because we may be called from a place where that
```

Comment:
Why does this comment say "Can't use pTHX" if the function now uses `pTHX`?

Reply:
Because I made a mistake; now fixed